Ethernet/W5100S/w5100s.h

```diff
@@ -3087,7 +3087,7 @@ uint16_t getSn_RX_RSR(uint8_t sn);
  * @return uint16_t. Max buffer size
  */
 #define getSn_RxMAX(sn) \
-		((uint16_t)(getSn_RXMEM_SIZE(sn)) << 10)
+		((uint16_t)(0x0001 << getSn_RXMEM_SIZE(sn)) << 10)
```

Comment:
About 3090 and 3110 line, You're right.
If you apply as you commented us, a return value is 1024,2048,4096, 8192.
Those line will change  as you commented us.
Could you please send the pull request the changed code again?